src/PKSim.Infrastructure/Serialization/Xml/Serializers/ApplicationSettingsXmlSerializer.cs

```diff
@@ -14,6 +14,7 @@ public ApplicationSettingsXmlSerializer()
 
       public override void PerformMapping()
       {
+         Map(x => x.MoBiPath);
```

Comment:
User is required to set this path before using MoBi?
Does the suite come with both apps zipped together as well?